{
  try {
    testCreateAndRecreate(false);
    assertListenerStatus();
  }
 catch (  SubscriptionUserApiException e) {
    log.error("Unexpected exception",e);
    Assert.fail(e.getMessage());
  }
}
